test/src/com/xilinx/rapidwright/design/TestDesignTools.java

```diff
@@ -115,6 +118,52 @@ public void testCopyImplementationRouteThruVCCPinCheck() {
         }
     }
 
+
+    private void testCopyImplementationHelper(boolean keepStaticRouting, HashMap<String, Integer> numPIPs) {
+        String dcpPath = RapidWrightDCP.getString("hwct_pr1.dcp");
```

Comment:
Did you mean to change this to `hwct_pr1.dcp`? The diff with #435 is this:
```
$ git diff copy_static_nets -- test/src/com/xilinx/rapidwright/design/TestDesignTools.java
diff --git a/test/src/com/xilinx/rapidwright/design/TestDesignTools.java b/test/src/com/xilinx/rapidwright/design/TestDesignTools.java
index e9db308..e433c06 100644
--- a/test/src/com/xilinx/rapidwright/design/TestDesignTools.java
+++ b/test/src/com/xilinx/rapidwright/design/TestDesignTools.java
@@ -120,7 +120,7 @@ public class TestDesignTools {
 
 
     private void testCopyImplementationHelper(boolean keepStaticRouting, HashMap<String, Integer> numPIPs) {
-        String dcpPath = RapidWrightDCP.getString("testCopyImplementation.dcp");
+        String dcpPath = RapidWrightDCP.getString("hwct_pr1.dcp");
         String srcCellName = "clock_isolation";
 
         Design src = Design.readCheckpoint(dcpPath);
@@ -147,8 +147,8 @@ public class TestDesignTools {
         boolean keepStaticRouting = true;
         HashMap<String, Integer> numPIPs = new HashMap<String, Integer>()
         {{
-            put("GLOBAL_LOGIC0",  201);
-            put("GLOBAL_LOGIC1",  601);
+            put("GLOBAL_LOGIC0",  218);
+            put("GLOBAL_LOGIC1",  604);
         }};
 
         testCopyImplementationHelper(keepStaticRouting, numPIPs);
```

Reply:
You shouldn't be changing this test at all?

Reply:
revert to use testCopyImplementation.dcp